Simplenote/NoteTableCellView.swift

```diff
@@ -0,0 +1,161 @@
+import Foundation
+import AppKit
+
+
+// MARK: - TagTableCellView
+//
+@objcMembers
+class NoteTableCellView: NSTableCellView {
+
+    /// TextField: Title
+    ///
+    @IBOutlet private var titleTextField: NSTextField!
+
+    /// TextField: Body
+    ///
+    @IBOutlet private var bodyTextField: NSTextField!
+
+    /// LeftImage: Pinned Indicator
+    ///
+    @IBOutlet private var leftImageView: NSImageView!
+
+    /// RightImage: Shared Indicator
+    ///
+    @IBOutlet private var rightImageView: NSImageView!
+
+    /// Indicates if the receiver displays the pinned indicator
+    ///
+    var displaysPinnedIndicator: Bool {
+        get {
+            !leftImageView.isHidden
+        }
+        set {
+            leftImageView.isHidden = !newValue
+        }
+    }
+
+    /// Indicates if the receiver displays the shared indicator
+    ///
+    var displaysSharedIndicator: Bool {
+        get {
+            !rightImageView.isHidden
+        }
+        set {
+            rightImageView.isHidden = !newValue
+        }
+    }
+
+    /// In condensed mode we simply won't render the bodyTextField
+    ///
+    var rendersInCondensedMode: Bool {
+        get {
+            bodyTextField.isHidden
+        }
+        set {
+            bodyTextField.isHidden = newValue
+        }
+    }
+
+    /// Note's Title String
+    ///
+    var titleString: String? {
```

Comment:
Again nitpicking, but this could just be `title` or `noteTitle`? We know it's a string because of the type :) 

(same for `bodyString` below)

Reply:
You're SO RIGHT!!! What was I thinking about!!!